### PR TITLE
Feature/linting in pipeline

### PR DESCRIPTION
**Pull request by BradleyCulley**
Also added .idea folder to git ignore file

### CODE REVIEW

**Comment by adamrturman** [comment]
Let's replace "and" with "&" for continuity.

File: .github/workflows/github_action.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   test:
-    name: Running Units Tests
+    name: Running Units Tests and Linting
```
Comment (adamrturman) [inline .github/workflows/github_action.yml]: Have you considering changing "and" to "&?"

**Review by adamrturman — CHANGES_REQUESTED** [review]
See comment regarding "&."

File: .github/workflows/github_action.yml (hunk 1 above)
Comment (BradleyCulley) [reply .github/workflows/github_action.yml]: Sounds good, I'm on it...

**Comment by BradleyCulley** [comment]
Just made the requested change around "&". Just let me know if anything else needed, or it wasn't what you wanted...

File: .github/workflows/github_action.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   test:
-    name: Running Units Tests and Linting
+    name: Running Units Tests & Linting
```
Comment (adamrturman) [inline .github/workflows/github_action.yml:9]: Looks good!

**Review by adamrturman — APPROVED** [review]

**Pull request merged**